fractal_server/app/routes/api/v1/project.py

```diff
@@ -395,21 +395,28 @@ async def apply_workflow(
         workflow_id=workflow_id,
         user_email=user.email,
         input_dataset_dump=dict(
-            input_dataset.model_dump(exclude={"resource_list"}),
+            input_dataset.model_dump(
+                exclude={"resource_list", "timestamp_created"}
+            ),
+            timestamp_created=str(input_dataset.timestamp_created),
             resource_list=[
                 resource.model_dump()
                 for resource in input_dataset.resource_list
             ],
         ),
         output_dataset_dump=dict(
-            output_dataset.model_dump(exclude={"resource_list"}),
+            output_dataset.model_dump(
+                exclude={"resource_list", "timestamp_created"}
+            ),
+            timestamp_created=str(output_dataset.timestamp_created),
             resource_list=[
                 resource.model_dump()
                 for resource in output_dataset.resource_list
             ],
         ),
         workflow_dump=dict(
-            workflow.model_dump(exclude={"task_list"}),
+            workflow.model_dump(exclude={"task_list", "timestamp_created"}),
+            timestamp_created=str(workflow.timestamp_created),
```

Comment:
This way of serializing `timestamp_created` is fine, but it doesn't match with what we do for `project_dump` a few lines below. Can we make them homogeneous?
The current approach (exclude, and then use `str`) looks simpler than `json.loads(project.json(...))`, thus I'm OK with having it also for `project_dump` - unless I'm missing some other issues.